src/NUnitEngine/nunit.engine/Internal/RuntimeFrameworks/DotNetFrameworkLocator.cs

```diff
@@ -1,3 +1,26 @@
+// ***********************************************************************
+// Copyright (c) 2007 Charlie Poole, Rob Prouse
```

Comment:
Shouldn't this be 2018, since this is a new file, although the logic is moved from an existing file.

Reply:
I thought we kept the original date when code was just moved? I am absolutely not an authority on this stuff though!

Reply:
Could be either, I don't think we have a hard and fast rule. At my company, if we make substantial changes, we update the copyright to the current year.